test/others/test_schema.py

```diff
@@ -1,55 +1,88 @@
-from haystack.schema import Document, Label, Answer, Span, MultiLabel
+from haystack.schema import Document, Label, Answer, Span, MultiLabel, TableCell
 import pytest
 import numpy as np
 import pandas as pd
 
 from ..conftest import SAMPLES_PATH, fail_at_version
 
-LABELS = [
-    Label(
-        query="some",
-        answer=Answer(
-            answer="an answer",
-            type="extractive",
-            score=0.1,
-            document_ids=["123"],
-            offsets_in_document=[Span(start=1, end=3)],
+
+@pytest.fixture
+def text_labels():
```

Comment:
These are not new changes, just made LABELS a fixture instead of a global variable.